### PR TITLE
docs: draft on improving commit context

### PR DESCRIPTION
**Pull request by ijsong**
### What this PR does

RFC draft on improving commit context

### Which issue(s) this PR resolves

Ref: #125

### Anything else

- - -
This change is [Reviewable](https://reviewable.io/reviews/kakao/varlog/149)

### CODE REVIEW

File: docs/RFCs/20220915_commit_context.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+- Status: draft
+- Authors: @ijsong
+- Varlog Issue: #125
+- RFC PR:
+
+# Summary
+
+A large amount of empty commit contexts not only causes too slow recovery of
+log streams but also takes a quite large disk space. To solve these problems,
+this RFC proposes to store only the latest non-empty commit context rather
```
Comment (hungryjang) [inline docs/RFCs/20220915_commit_context.md:11]: I think we need latest commit context & latest non-empty commit context both.
Latest commit context is used for valid report version to trim commit result in metadata repository

Comment (ijsong) [reply docs/RFCs/20220915_commit_context.md:11]: Yes, you're right. I'll fix it.

Comment (ijsong) [reply docs/RFCs/20220915_commit_context.md]: @hungryjang 
Modified the draft to store the latest commit context regardless of emptiness.

**Review by hungryjang — APPROVED** [review]

**Comment by ijsong** [comment]
@hungryjang 
I modified a hyperlink after your approval.
https://github.com/ijsong/varlog/blob/84047c5125ccc1f7f493cf352fa3f8c1399d485f/docs/RFCs/20220915_commit_context.md?plain=1#L245

**Pull request merged**